docs/websockets.md

```diff
@@ -109,3 +109,17 @@ correctly updated.
 
 * `await websocket.send(message)`
 * `await websocket.receive()`
+
+### Rejecting the connection
+
+Before the calling `websocket.accept()` it is possible to reject the connection,
+e.g. because of failing authentication.  The standard way to do this
+is to call `websocket.close()` which will cause the ASGI web server
+to respond to the initator with a HTTP 403 error.
+It is also possible to construct a `Response` object and send that:
+
+* `await websocket.send_response(response)`
+
+If the ASGI server supports the `websocket.http.response` extension, this
+will cause the specified response to be sent to the initiator, before closing
+the connection.  Otherwise, it behaves like `websocket.close()`.
```

Comment:
```suggestion
the connection.
```

Reply:
See my other comment.  I think it is useful for the application to provide a custom response, if the server provides it, otherwise it provides the only response, possible, the close().    I don't really see what benefit there is to moving that logic to the application, since there is only one course of action.